scanner/image/db/postgresql.conf

```diff
@@ -0,0 +1,755 @@
+# -----------------------------
```

Comment:
This should not be stored in the image and instead should be part of the helm templates as a config map otherwise it's basically unchangeable and that has generally been a pain for us